tests/integration/components/image-drop-test.js

```diff
@@ -75,7 +75,7 @@ test('it handles a dropped image file', function(assert) {
   this.$('.image-drop').trigger('dragover');
   assert.equal(this.$('.image-drop').hasClass('active'), true);
 
-  fillInFileInput('input', { name: fileName, content: droppedImageString });
+  fillInFileInput('input[type=file]', { name: fileName, content: droppedImageString });
```

Comment:
Just `input` gives out hundreds of results, due to the testing UI, so this should boost performance in a tiny way. Not really that important, though.